airflow/sensors/sql.py

```diff
@@ -90,7 +94,7 @@ def _get_hook(self):
                 f"Connection type ({conn.conn_type}) is not supported by SqlSensor. "
                 + f"Supported connection types: {list(allowed_conn_type)}"
             )
-        return conn.get_hook()
+        return conn.get_hook(self.hook_params)
```

Comment:
```suggestion
        return conn.get_hook(hook_params=self.hook_params)
```

Using a keyword argument might make this more readable and less likely to break if someone later adds another argument to the `get_hook` function.

Reply:
Definitely agree with you